src/Utilities/SeekCamera/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+# Developed by MACBIOIDI-ULPGC & IACTEC-IAC group
+CMAKE_MINIMUM_REQUIRED(VERSION 3.3.0)
+
+PROJECT(SeekCameraLib)
+
+SET(${PROJECT_NAME}_INCLUDE_DIRS
+  ${CMAKE_CURRENT_SOURCE_DIR}
+  ${CMAKE_CURRENT_BINARY_DIR}
+  )
+
+  SET(${PROJECT_NAME}_SRCS
+   SeekCam.cpp
+   SeekDevice.cpp
+   SeekThermal.cpp
+   SeekThermalPro.cpp
+  )
+
+  SET(${PROJECT_NAME}_HDRS
+   seek.h
+   SeekCam.h
+   SeekDevice.h
+   SeekLogging.h
+   SeekThermal.h
+   SeekThermalPro.h
+  )
+
+  
+ # libusb-1.0 -----
+find_path(LIBUSB_INCLUDE_DIR
+  NAMES libusb.h
+  PATH_SUFFIXES "include" "libusb" "libusb-1.0"
+)
+
+find_library(LIBUSB_LIBRARY
+  NAMES usb-1.0
+  PATH_SUFFIXES "lib" "lib32" "lib64"
+)
+
+if (NOT LIBUSB_LIBRARY)
+    message(FATAL_ERROR "Libusb library not found")
+endif()
+
+FIND_PACKAGE(OpenCV REQUIRED PATHS ${OpenCV_DIR})
+
+if (NOT OpenCV_FOUND)

```

Comment:
FIND_PACKAGE REQUIRED will do this error message for you.

Reply:
Ok. Thank you.